scripts/apps/authoring/authoring/authoring-custom-field.tsx

```diff
@@ -9,6 +9,7 @@ interface IProps {
     field: IArticleField;
     editable: boolean;
     onChange: (field: IArticleField, value: any) => any;
+    template: any;
```

Comment:
Can you specify the type? It looks like there is no interface yet. You should add one in that case.